### PR TITLE
Minor fixes from SAT-9421

### PR DESCRIPTION
**Pull request by mjivraja**
Procedures in section 3.4 are now visible.

https://issues.redhat.com/browse/SAT-9421

Cherry-pick into:

* [X] Foreman 3.3
* [X] Foreman 3.2
* [X] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

**Comment by mjivraja** [comment]
Hi @evgeni 

I have taken the change as per your comment https://github.com/theforeman/foreman-documentation/pull/1279/files#r890955941

Procedures in Section 3.4 is now visible.

**Review by evgeni — CHANGES_REQUESTED** [review]
That doesn't look correct to me.

Foreman/Katello now renders weirdly, the index has C7, C8 RHEL7, but the section header for RHEL7 is missing:
image

You could argue that the steps are all identical and we can just use `yum` everywhere and then we just collapse *all* of these to "one OS", or we need 4 sections for Foreman/Katello and 2 for Satellite, where we now have one, without a heading?

image

Or maybe have two, one EL7 one EL8 with the "right" package manager, and call the sections "RHEL X" for Satellite and "CentOS / RHEL X" for Foreman/Katello?

**Comment by mjivraja** [comment]
Hi @evgeni 

I have taken the change as per your comment https://github.com/theforeman/foreman-documentation/pull/1279/files#r890955941

Procedures in Section 3.4 is now visible.


> Or maybe have two, one EL7 one EL8 with the "right" package manager, and call the sections "RHEL X" for Satellite and "CentOS / RHEL X" for Foreman/Katello?

I think this would be better. Can we use `dnf update` && `dnf install satellite` for RHEL 8 and CentOS 8?

**Comment by evgeni** [comment]
Satellite doesn't exist for CentOS.

But proc_installing-server-packages-el.adoc already renders the correct package name (satellite / foreman-installer / foreman-installer-katello) depending on the project we're building for.

**Comment by maximiliankolb** [comment]
@mjivraja Pleae change the title for your PR. Community users have to access to SAT tickets > please make it more descriptive.

**Comment by mjivraja** [comment]
> @mjivraja Pleae change the title for your PR. Community users have to access to SAT tickets > please make it more descriptive.

Noted @maximiliankolb. I will keep it descriptive.

**Comment by melcorr** [comment]
> > @mjivraja Pleae change the title for your PR. Community users have to access to SAT tickets > please make it more descriptive.
> 
> Noted @maximiliankolb. I will keep it descriptive.

It is best to never reference the Jira tickets on GitHub. They are inaccessible to the community. 
I am OK with some number in the branch name for your references, but everywhere else should probably be with the open source community in mind.

**Comment by mjivraja** [comment]
> Satellite doesn't exist for CentOS.
> 
> But proc_installing-server-packages-el.adoc already renders the correct package name (satellite / foreman-installer / foreman-installer-katello) depending on the project we're building for.

I have restructured as per your review. Please have a look at the latest commit.

File: guides/common/modules/proc_installing-the-satellite-server-packages.adoc [hunk 1]
```diff
@@ -3,38 +3,46 @@ ifdef::context[:parent-context: {context}]
...
+endif::[]
 ifdef::foreman-el,katello[]
 * xref:#centos-8[CentOS 8]
 * xref:#centos-7[CentOS 7]
 endif::[]
-ifdef::foreman-el,katello[]
-* xref:#rhel-7[{RHEL} 7]
-endif::[]
 
 .Procedure
```
Comment (evgeni) [inline guides/common/modules/proc_installing-the-satellite-server-packages.adoc:15]: Unrelated to this PR, but popped up in my head.

This "procedure" looks odd, as it is only rendered for the very first OS entry:
image

**Review by evgeni — APPROVED** [review]
Renders correctly now.

Added a small thing in the middle, but it's unrelated to the actual changes done in this PR.

**Pull request merged**